client/src/components/CustomFields.js

```diff
@@ -286,9 +286,42 @@ const EnumField = fieldProps => {
 
 const enumHumanValue = (choices, fieldVal) => {
   if (Array.isArray(fieldVal)) {
-    return fieldVal && fieldVal.map(k => choices[k]?.label).join(", ")
+    return (
+      <div>
+        {fieldVal
+          .map(k => (
+            <Badge
+              key={k}
+              style={{
+                fontSize: "inherit",
+                fontWeight: "inherit",
+                lineHeight: "inherit",
+                color: "black",
+                backgroundColor: choices[k]?.color
+              }}
+            >
+              {choices[k]?.label}
+            </Badge>
+          ))
+          .reduce((acc, x) => (acc === null ? [x] : [...acc, ", ", x]), null)}
```

Comment:
I think using Array.reduce() for adding a comma seperator is a bit expensive. The addition of comma can take place during the first iteration (Array.map) like the following:
```
{fieldVal.map((k, index) => (
  <>
    <Badge
      key={k}
      style={{
        fontSize: "inherit",
        fontWeight: "inherit",
        lineHeight: "inherit",
        color: "black",
        backgroundColor: choices[k]?.color
      }}
    >
      {choices[k]?.label}
    </Badge>
    {index < fieldVal.length - 1 && " , "}
  </>
))}
```